apps/src/templates/manageStudents/manageStudentsRedux.js

```diff
@@ -46,6 +47,7 @@ const blankNewStudentRow = {
   gender: '',
   username: '',
   loginType: '',
+  sharingDisabled: true,
```

Comment:
I think these should default to `false` since that's the server default for all students right now.

Reply:
I'm open to having my mind changed on this one though. It might be odd to have it pre-checked and disabled, but that's what it'll actually be on the server?